modules/gapi/perf/render/gapi_render_perf_tests_inl.hpp

```diff
@@ -0,0 +1,1144 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+
+#ifndef OPENCV_GAPI_RENDER_PERF_TESTS_INL_HPP
+#define OPENCV_GAPI_RENDER_PERF_TESTS_INL_HPP
+
+#include <iostream>
+
+#include "../common/gapi_render_perf_tests.hpp"
+
+#include "../../src/api/render_priv.hpp"
+
+namespace opencv_test
+{
+  using namespace perf;
+
+  namespace {
+    void create_rand_mats(const cv::Size& size, MatType type, cv::Mat& ref_mat, cv::Mat& gapi_mat) {
```

Comment:
Can we reuse function from `TestPerfParams` to init mats ?

Reply:
Yes we can
But in this case we will have to use mats named in_mat1 and in_mat2 instead y_in_gapi_mat y_out_gapi_mat
I think better way to use mats named y_in_gapi_mat y_out_gapi_mat for better code understanding